tide_landing_page.install

```diff
@@ -1655,3 +1655,112 @@ function tide_landing_page_update_8036() {
   $config_read = _tide_read_config($json_field, $config_location, TRUE);
   $config_storage->write($json_field, $config_read);
 }
+
+/**
+ * Replace featured_news with navigation card.
+ */
+function tide_landing_page_update_8037() {
+  $moduleHandler = \Drupal::service('module_handler');
+  if (!$moduleHandler->moduleExists('tide_news')) {
+    return;
+  }
+  $paragraph_query = \Drupal::entityTypeManager()
```

Comment:
Be consistent; do not mix camelCase and snake_case variable naming inside a file.

Reply:
Where is there camelCase variable naming?

Reply:
@anthony-malkoun $moduleHandler.

Reply:
cool, changed.